src/sympc/tensor/replicatedshare_tensor.py

```diff
@@ -123,14 +123,12 @@ def get_shares(self) -> List[torch.Tensor]:
     def sanity_checks(
         x: "ReplicatedSharedTensor",
         y: Union[int, float, torch.Tensor, "ReplicatedSharedTensor"],
-        op_str: str,
-    ) -> "ReplicatedSharedTensor":
+    ) -> tuple("ReplicatedSharedTensor",):
```

Comment:
MAYBE: `python3.8` does not have this feature to use basic types as type hints.
Use `Tuple` from the `typing` library

Reply:
+ Add all the results